modules/Dasboard/components/CreateClassroomButton/CreateClassroomButton.tsx

```diff
@@ -15,9 +20,11 @@ const CreateClassroomButton: React.FC = () => {
         leftIcon={<FaPlus size={14} />}
         className={classes.button}
         classNames={{ leftIcon: classes.icon }}
+        onClick={handleOpenModal}
       >
         Create a classroom
       </Button>
+      <ClassroomFomModal opened={isModalOpen} onClose={handleCloseModal} />
```

Comment:
[SUPER NIT]

```suggestion
      <ClassroomFormModal opened={isModalOpen} onClose={handleCloseModal} />
```

Reply:
Why was it working ? Even though the spelling was mistake

Reply:
The component name itself was likely wrong and then auto imports did its thing.